core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -216,6 +218,77 @@ class AdminClientIntegrationTest extends KafkaServerTestHarness with Logging {
   }
 
   @Test
+  def testDescribeLogDirs(): Unit = {
+    client = AdminClient.create(createConfig())
+    val topic = "topic"
+    val leaderByPartition = TestUtils.createTopic(zkUtils, topic, 10, 1, servers, new Properties())
+    val partitionsByBroker = leaderByPartition.groupBy(_._2).mapValues(_.keys.toSeq)
```

Comment:
Instead of using (_._2), could we use case to define named variables to make it clear? Ditto on line 233.

Reply:
Sure. I replaced them with the following:

```
val partitionsByBroker = leaderByPartition.groupBy { case (partitionId, leaderId) => leaderId }.mapValues(_.keys.toSeq)`

val replicaInfos = logDirInfos.asScala.flatMap { case (logDir, logDirInfo) => logDirInfo.replicaInfos.asScala }.filterKeys(_.topic == topic)
```